### DISCUSSION TITLE
Users of a GitHub action are sometimes not shown in the main page of a repository

### DISCUSSION THREAD

**Discussion by JJ** [community/community#124960] [Actions] [answered]
### Select Topic Area

Question

### Body

In GitHub actions such as [this one](https://github.com/JJ/github-pr-contains-action), sometimes I see a "used by" sidebar that shows how many other repos are using an action. However, it disappears in other occasions and I don't really get what's the trigger that makes it show up. Any idea?

**Comment by nithinlycanz** [comment, marked as answer]
Actually it just shows that repo's that uses your specific github action. Idk why sometimes it's not shown, but it is a feature showing the popularity of your repo.

**Reply by JJ** [reply]
I know it's also available from the dependencies tab in Insights, but still, it's kind of surprising.

**Reply by nithinlycanz** [reply]
yeaah feel free to mark answer.
